class RetryCountOverError(Exception):
    """リトライ上限に達した"""
    ...